pkg/vault/test/authorizer/TimelockAuthorizerRoot.test.ts

```diff
@@ -74,11 +74,9 @@ describe('TimelockAuthorizer root', () => {
 
           it('emits an event', async () => {
             const newPendingRoot: SignerWithAddress = getNewPendingRoot();
-            const id = await authorizer.scheduleRootChange(newPendingRoot, [], { from: root });
+            const receipt = await authorizer.instance.connect(root).scheduleRootChange(newPendingRoot.address, []);
 
-            await advanceTime(ROOT_CHANGE_DELAY);
-            const receipt = await authorizer.execute(id);
-            expectEvent.inReceipt(await receipt.wait(), 'PendingRootSet', { pendingRoot: newPendingRoot.address });
+            expectEvent.inReceipt(await receipt.wait(), 'RootChangeScheduled', { newRoot: newPendingRoot.address });
```

Comment:
This new test is correct, but let's also keep the old test (i.e. that `PendingRootSet` is emitted correctly when executing). Sorry if my previous message was not clear.